packages/inputs-minimal/src/select/index.jsx

```diff
@@ -0,0 +1,98 @@
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { SIZES, SIZES_KEYS } from '@/src/constants';
+import { ReactComponent as ArrowIcon } from './arrow.svg';
+import Meta from '../textfields_meta';
+import styles from './index.module.scss';
+
+const Select = ({
+  label,
+  error,
+  value = '',
+  options,
+  size = SIZES.medium,
```

Comment:
Not sure if we have a naming convention for 'enums'. I would have used `Sizes.medium` - have no strong opinion about it though but we should stick to one scheme